### PR TITLE
Improved display of log types.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Refactored label displays for log types to capitalize text.

### Issues Resolved
[List any issues this PR will resolve]

##### Before
Screenshot 2023-10-12 at 1 11 17 PM
##### After
Screenshot 2023-10-12 at 1 17 35 PM
##### Before
Screenshot 2023-10-12 at 1 11 39 PM
##### After
Screenshot 2023-10-12 at 1 17 16 PM
##### Before
Screenshot 2023-10-12 at 1 10 58 PM
##### After
Screenshot 2023-10-12 at 1 08 29 PM

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AWSHurneyt** [comment]
Haven't been able to reproduce locally the `3_alerts.spec.js` `4_findings.spec.js` cypress test failures we're seeing in this github CI; but all cypress tests are passing locally on a 3.0 cluster. 
```
====================================================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  1_detectors.spec.js                      02:25       13       13        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  2_rules.spec.js                          03:55       16       16        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  3_alerts.spec.js                         01:55        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  4_findings.spec.js                       01:42        6        5        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  5_integrations.spec.js                   01:06        2        -        -        2        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        11:07       45       42        -        3        -  

✨  Done in 851.48s.

**Comment by AWSHurneyt** [comment]
@amsiglan confirmed that the `3_alerts.spec.js` and `4_findings.spec.js` cypress test failures are due to the backend plugin; they're **not** related to this PR.

**Review by eirsep — APPROVED** [review]

File: public/pages/LogTypes/utils/constants.ts [hunk 1]
```diff
@@ -23,3 +23,21 @@ export const defaultLogType: LogTypeBase = {
   tags: null,
   category: '',
 };
+
+export const logTypeLabels = {
+  cloudtrail: 'Cloudtrail',
+  dns: 'DNS',
+  vpcflow: 'VPC Flow',
+  ad_ldap: 'Ad/ldap',
```
Comment (eirsep) [inline public/pages/LogTypes/utils/constants.ts:31]: `AD/LDAP` ?

Comment (AWSHurneyt) [reply public/pages/LogTypes/utils/constants.ts:31]: On the `Log types` table page, the description for that type is `Ad/ldap logs`, so I went with that capitalization pattern.

Comment (eirsep) [reply public/pages/LogTypes/utils/constants.ts:31]: got it

Comment (AWSHurneyt) [reply public/pages/LogTypes/utils/constants.ts:31]: I added screenshots to the PR overview, and here's a screenshot with the descriptions for reference.
Screenshot 2023-10-12 at 1 24 35 PM

Comment (AWSHurneyt) [reply public/pages/LogTypes/utils/constants.ts]: Adjusted these labels based on conversation with Amardeep.

**Review by amsiglan — APPROVED** [review]

**Pull request merged**